### PR TITLE
check comments

### PR DESCRIPTION
**Pull request by newbazz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,9 +16,13 @@
   "dependencies": {
     "babel-core": "^6.26.0",
     "babel-preset-node8": "^1.2.0",
+    "body-parser": "^1.18.2",
```
Comment (meghprkh) [inline package.json]: Remove all these

File: src/controllers/scoreboard.js [hunk 2]
```diff
@@ -6,4 +6,4 @@ export async function get(ctx) {
       ['score', 'DESC']],
     attributes: { exclude: [ 'email' ] }
   })
-}
```
Comment (meghprkh) [inline src/controllers/scoreboard.js]: Why this change? Is editorconfig not installed? However go to your editor settings and add line at end of file

Comment (meghprkh) [reply src/controllers/scoreboard.js]: However this is not for this pr, just in general

File: src/models/Comment.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    }
+  },  
+  comment: {
+    type: Sequelize.STRING,
+    allowNull: true,
+    validate: {
+      notEmpty: false,
+    }
+  },
+  time: {
```
Comment (meghprkh) [inline src/models/Comment.js]: Remove this, no need since sequelize stores it by default

File: src/models/Comment.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import Sequelize from 'sequelize';
+import db from './db'
+
+const Comment = db.define('comment', {
+  qno: {
```
Comment (meghprkh) [inline src/models/Comment.js]: Dont store it like this use relations http://docs.sequelizejs.com/manual/tutorial/associations.html#belongsto

File: src/routes/question.js [hunk 5]
```diff
@@ -30,7 +30,7 @@ const routes = [
       },
       type: 'form',
       body: {
-        answer: Joi.string(),
+        answer:Joi.string(),
```
Comment (meghprkh) [inline src/routes/question.js]: Add back the space :p . Dont edit unnecessary files

File: src/controllers/Comment.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import Comment from '../models/Comment'
+
+export async function addComment(ctx) {
+  const { qno } = ctx.params
+  const { user } = ctx.state
+  const comment= ctx.request.body.comment
+  await Comment.create({qno: qno, comment: comment, time: Date(), umail: user.email})
```
Comment (meghprkh) [inline src/controllers/Comment.js]: Use relations for both user and question

**Review by meghprkh — CHANGES_REQUESTED** [review]

File: src/models/Comment.js (hunk 4 above)
Comment (meghprkh) [reply src/models/Comment.js]: Sorry http://docs.sequelizejs.com/manual/tutorial/associations.html#one-to-many-associations

File: src/models/Comment.js (hunk 3 above)
Comment (newbazz) [reply src/models/Comment.js]: but the issue says to store the time stamp

Comment (meghprkh) [reply src/models/Comment.js]: It is automatically stored by sequelize

**Review by meghprkh — APPROVED** [review]

File: src/controllers/question.js [hunk 7]
```diff
@@ -37,3 +38,12 @@ export async function getAll(ctx) {
   })
   // XXX: Include only question number and title and nothing else
 }
+export async function addcomment(ctx) {
```
Comment (ironmaniiith) [inline src/controllers/question.js:41]: You should use camelCase for this function name `addComment`
